schedule/yast/autoyast_create_hdd_gnome_RAID_encrypted_home.yaml

```diff
@@ -0,0 +1,22 @@
+name: autoyast_create_hdd_gnome_RAID_encrypted_home
+description:    >
+  Test performs autoyast installation to test encrypted home partition on RAID.
+conditional_schedule:
```

Comment:
no need for this, only s390x for now.

Reply:
Removed the conditional schedule, it is not needed since we won't publish qcow for the test.